django_tables2/export/export.py

```diff
@@ -41,16 +44,25 @@ class TableExport:
         YAML: "text/yml; charset=utf-8",
     }
 
-    def __init__(self, export_format, table, exclude_columns=None):
+    def __init__(self, export_format, table, exclude_columns=None, dataset_kwargs=None):
         if not self.is_valid_format(export_format):
             raise TypeError('Export format "{}" is not supported.'.format(export_format))
 
         self.format = export_format
-        self.dataset = self.table_to_dataset(table, exclude_columns)
+        self.dataset = self.table_to_dataset(table, exclude_columns, dataset_kwargs)
 
-    def table_to_dataset(self, table, exclude_columns):
+    def table_to_dataset(self, table, exclude_columns, dataset_kwargs=None):
         """Transform a table to a tablib dataset."""
-        dataset = Dataset()
+
+        def default_dataset_title():
+            try:
+                return table.Meta.model._meta.verbose_name_plural.title()
+            except AttributeError:
+                return "Export Data"
+
+        dataset_kwargs = dataset_kwargs or {}
+        dataset_kwargs["title"] = dataset_kwargs.get("title") or default_dataset_title()
```

Comment:
For these defaults-overridden by supplied dict, I prefer to make that a little more explicit by defining the defaults first, and then update the dictionary with the customized values:

```
kwargs = {"title": default_dataset_title()}
kwargs.update(dataset_kwargs or {})
dataset = Dataset(**kwargs)
```

Reply:
Good improvement.  Thanks.  Done.